### PR TITLE
[Feature] - Change namespace from Ourserverdev to IronWeb in all the bundle.

### PR DESCRIPTION
**Pull request by lechatquidanse**
# Feature

#### What's this PR do ?

Update namespace to follow github organization name.

- update namespace
- update composer.json
- update readme

### CODE REVIEW

File: Tests/boostrap.php [hunk 1]
```diff
@@ -8,9 +8,9 @@
 require_once $file;
 
 use Doctrine\DBAL\Types\Type;
-use Ourserverdev\DoctrineBundle\DBAL\Types\DateIntervalType;
+use IronWeb\DoctrineBundle\DBAL\Types\DateIntervalType;
 
 Type::addType(
     DateIntervalType::DATE_INTERVAL_TYPE_NAME,
-    'Ourserverdev\DoctrineBundle\\DBAL\\Types\\DateIntervalType'
+    'IronWeb\DoctrineBundle\\DBAL\\Types\\DateIntervalType'
```
Comment (jfacchini) [inline Tests/boostrap.php]: `\` or `\\` ?

**Comment by jfacchini** [comment]
Otherwise :+1:

**Comment by jfacchini** [comment]
Maybe we can improve this line https://github.com/iron-web/doctrine-bundle/blob/master/DBAL/Types/DateIntervalType.php#L47 by using
```php
if ($value instanceof \DateInterval) {
```
And then remove the comment under this line.

**Comment by lechatquidanse** [comment]
@jfacchini updated

**Comment by jfacchini** [comment]
Great, :+1: and merge if the tests passed

**Pull request merged**